scripts/release/create-canary-commands/build-artifacts.js

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const {exec} = require('child-process-promise');
+const {join} = require('path');
+const {logPromise} = require('../utils');
+
+const run = async ({cwd, dry, tempDirectory}) => {
+  const defaultOptions = {
+    cwd: tempDirectory,
+  };
+
+  await exec('yarn install', defaultOptions);
+  await exec('yarn build -- --extract-errors', defaultOptions);
+
+  const tempNodeModulesPath = join(tempDirectory, 'build', 'node_modules');
+  const buildPath = join(cwd, 'build');
+
+  await exec(`cp -r ${tempNodeModulesPath} ${buildPath}`);
+};
+
+module.exports = async params => {
+  return logPromise(run(params), 'Building artifacts', true);
```

Comment:
this takes a looong time. thank you for the spinners (really, at least I'm certain it hasn't 'hung'), but is there a reason the output is hidden? it's been about 5 mins for me now.

Reply:
> is there a reason the output is hidden?

Makes the overall flow of the build script easier to follow.

I guess if you think it's important to show the build script output, the scripts could be tweaked to do this. I don't feel _very_ strongly here.

Reply:
I don't feel strongly about this either. I wish I could add priorities to my comments. this is definitely lowest pri.